wifiphisher/common/opmode_manager.py

```diff
@@ -0,0 +1,269 @@
+                'characters.')
+
+        if args.handshake_capture and not os.path.isfile(
+                args.handshake_capture):
+            sys.exit('[' +
+                     constants.R +
+                     '-' +
+                     constants.W +
+                     '] handshake capture does not exist.')
+        elif args.handshake_capture and not handshakeverify.\
+                is_valid_handshake_capture(args.handshake_capture):
+            sys.exit('[' +
+                     constants.R +
+                     '-' +
+                     constants.W +
+                     '] handshake capture does not contain valid handshake')
+
+        if ((args.jamminginterface and not args.apinterface) or
+                (not args.jamminginterface and args.apinterface)) and \
+                not (args.nojamming and args.apinterface):
+            sys.exit(
+                '[' +
+                constants.R +
+                '-' +
+                constants.W +
+                '] --apinterface (-aI) and --jamminginterface (-jI)'
+                '(or --nojamming (-nJ)) are used in conjuction.')
+
+        if args.nojamming and args.jamminginterface:
+            sys.exit(
+                '[' +
+                constants.R +
+                '-' +
+                constants.W +
+                '] --nojamming (-nJ) and --jamminginterface (-jI)'
+                'cannot work together.')
+
+        if args.lure10_exploit and args.nojamming:
+            sys.exit(
+                '[' +
+                constants.R +
+                '-' +
+                constants.W +
+                '] --lure10-exploit (-lE) and --nojamming (-nJ)'
+                'cannot work together.')
+
+        if args.lure10_exploit and not os.path.isfile(
+                constants.LOCS_DIR + args.lure10_exploit):
+            sys.exit('[' +
+                     constants.R +
+                     '-' +
+                     constants.W +
+                     '] Lure10 capture does not exist. Listing directory'
+                     'of captures: ' + str(os.listdir(constants.LOCS_DIR)))
+
+        if (args.mac_ap_interface and args.no_mac_randomization) or \
+                (args.mac_deauth_interface and args.no_mac_randomization):
+            sys.exit(
+                '[' +
+                constants.R +
+                '-' +
+                constants.W +
+                '] --no-mac-randomization (-iNM) cannot work together with'
+                '--mac-ap-interface or --mac-deauth-interface (-iDM)')
+
+        # if args.deauth_essid is set we need the second card to
+        # do the frequency hopping
+        if args.deauth_essid and self._is_one_phy_interface:
+            print(
+                '[' +
+                constants.R +
+                '!' +
+                constants.W +
+                '] system has only one card and -dE will only deauth on '
+                'the target ap channel')
```

Comment:
How about "Only one card was found. Wifiphisher will deauth only on the target AP channel."?